releasenotes/notes/revert-yaml-in-secret-a2f8b0b7fbe6a820.yaml

```
@@ -0,0 +1,6 @@
+---
+issues:
```

`issues` is for known issues affecting this release. I think we don't have a choice other than using the `fixes` section instead.

```suggestion
fixes:
```